index.md

```diff
@@ -88,13 +88,18 @@ Make your event and your presentations accessible so that you are prepared for s
 
 Presentations that are accessible to people with disabilities are also **inclusive** to many more audiences. For example, people with different learning styles and people who are not fluent in the language.
 
-Accessible presentations also have additional benefits, such as in these situations:
+Accessible presentations also have additional benefits, such as in this situation:
 
--  A live presentation with visuals is recorded and made available online as an audio podcast. During the presentation you describe the visuals. That provides the visual information to people who are blind or can't see the visuals well during the presentation. People listening to the podcast later will also get the visual information.
--  [CART](#cart){:.termref} provides real-time text of the speaker and other audio. People who are deaf or hard of hearing use CART. CART also helps people who understand text better than spoken language. For example, people whose native language is different. CART output can also be used to develop a transcript.
--  Transcripts online can increase search engine optimization (SEO) for audio and video. [Transcripts also have many other benefits {% include_cached external.html %}](http://www.uiaccess.com/transcripts/transcripts_on_the_web.html).
+-  A live presentation with “slides” is recorded and made available online as an audio podcast. The presenter describe the important visual information in the slides during the presentation.
+   - That provides the visual information during the presentation to people who are blind and to people who cannot see the slides well.
+   - Later, people listening to the podcast also get the visual information that was described, even though they don’t have video.
+-  During the presentation, a professional captioner types what is being said as text.
+   - People who are deaf or hard of hearing read the captioning.
+   - It also helps people who understand written text better than spoken language. For example, people whose native language is different.
+   - Later, the live captioning output is used to develop a text transcript of the presentation.
+-  That transcript is put online and it increases search engine optimization (SEO) for the presentation and your website.
```

Comment:
Same here, for me personally it is difficult to follow this in such bulleted format.